src/meta/api/endpoint/ApiEndPoint.ts

```diff
@@ -108,6 +108,7 @@ export const ApiEndPoint = {
       forest: () => apiPath('geo', 'estimations', 'forest'),
       forestAgreement: () => apiPath('geo', 'estimation', 'forestAgreement'),
     },
+    bounds: () => apiPath('geo', 'bounds'),
```

Comment:
better to move it before sepalProxy, so we can keep it tidy :)
(Have Geo.<path> before Geo.<Key>.<path>)